src/test/resources/application-test.yml

```diff
@@ -75,7 +75,8 @@ courtsentencing:
   has-migrated-data: true
 contactperson:
    complete-check:
-     delay-seconds: 1
+     delay-seconds: 0
+     retry-seconds: 0
      count: 1
```

Comment:
experiment to speed up test - which seems to work since we only migrate 2 contacts in our test